locales/sk/json.json

```diff
@@ -448,32 +448,32 @@
     "Niger": "Niger",
     "Nigeria": "Nigéria",
     "Niue": "Niue",
-    "No": "Žiadny",
-    "No :resource matched the given criteria.": "Č. :resource zodpovedali daným kritériám.",
-    "No additional information...": "Žiadne ďalšie informácie...",
-    "No Current Data": "Žiadne Aktuálne Údaje",
-    "No Data": "Žiadne Údaje",
-    "no file selected": "nie je vybratý žiadny súbor",
-    "No Increase": "Žiadne Zvýšenie",
+    "No": "Nie",
+    "No :resource matched the given criteria.": "Nenašli sa žiadne položky.",
+    "No additional information...": "Žiadne dodatočné informácie...",
+    "No Current Data": "Žiadne aktuálne dáta",
+    "No Data": "Žiadne dáta",
+    "no file selected": "nebol vybraný žiadny súbor",
+    "No Increase": "Žiadne zvýšenie",
     "No payment methods on file.": "Neevidujeme žiadne spôsoby platby.",
     "No payment scheduled.": "Nie je naplánovaná žiadna platba.",
-    "No Prior Data": "Žiadne Predchádzajúce Údaje",
-    "No Results Found.": "Neboli Nájdené Žiadne Výsledky.",
+    "No Prior Data": "Žiadne prioritné dáta",
+    "No Results Found.": "Neboli nájdené žiadne výsledky.",
     "Norfolk Island": "Ostrov Norfolk",
     "Northern Mariana Islands": "Severné Mariány",
     "Norway": "Nórsko",
     "Not Found": "Nenájdené",
     "Notifications": "Upozornenia",
     "Nova": "Nova",
-    "Nova User": "Nova Používateľ",
-    "November": "novembra",
+    "Nova User": "Nova používateľ",
+    "November": "November",
```

Comment:
The name also needs to be added to the `_excludes.json` file, otherwise the status will show up as requiring translation.

Reply:
Added